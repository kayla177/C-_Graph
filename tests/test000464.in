ENTITY V0I9E059 name_V0I9 journal
ENTITY 1Z03QOED name_1Z03 author
ENTITY DTYRB83P name_DTYR institution
ENTITY SAQ5WYFO name_SAQ5 journal
ENTITY HGKZA6ZY name_HGKZ institution
ENTITY PMUE2DAE name_PMUE paper
ENTITY 66BG46AZ name_66BG paper
ENTITY S3PR1PYE name_S3PR institution
ENTITY OWRYJXG9 name_OWRY author
ENTITY DDLUUB6Q name_DDLU institution
RELATIONSHIP 1Z03QOED cited_by DDLUUB6Q 3.9
RELATIONSHIP DDLUUB6Q cited_by SAQ5WYFO 2.8
RELATIONSHIP S3PR1PYE authored_by DTYRB83P 10.0
RELATIONSHIP DDLUUB6Q cited_by OWRYJXG9 2.8
RELATIONSHIP 66BG46AZ cited_by SAQ5WYFO 4.9
RELATIONSHIP S3PR1PYE affiliated_with HGKZA6ZY 9.6
RELATIONSHIP DDLUUB6Q collaborated_with PMUE2DAE 2.7
RELATIONSHIP V0I9E059 affiliated_with 66BG46AZ 0.8
RELATIONSHIP PMUE2DAE cited_by HGKZA6ZY 8.2
RELATIONSHIP OWRYJXG9 authored_by V0I9E059 1.1
RELATIONSHIP S3PR1PYE collaborated_with SAQ5WYFO 6.2
RELATIONSHIP 1Z03QOED collaborated_with OWRYJXG9 8.9
RELATIONSHIP S3PR1PYE published_in 66BG46AZ 6.0
RELATIONSHIP OWRYJXG9 collaborated_with HGKZA6ZY 8.1
RELATIONSHIP S3PR1PYE collaborated_with DDLUUB6Q 9.9
PATH DDLUUB6Q PMUE2DAE
PATH 1Z03QOED 66BG46AZ
HIGHEST
PRINT OWRYJXG9
HIGHEST
PRINT V0I9E059
PRINT V0I9E059
DELETE V0I9E059
DELETE S3PR1PYE
DELETE PMUE2DAE
EXIT
