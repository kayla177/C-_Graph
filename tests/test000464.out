success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
DDLUUB6Q S3PR1PYE HGKZA6ZY PMUE2DAE 27.7
1Z03QOED OWRYJXG9 HGKZA6ZY S3PR1PYE 66BG46AZ 32.6
1Z03QOED DTYRB83P 36.6
DDLUUB6Q V0I9E059 1Z03QOED HGKZA6ZY
1Z03QOED DTYRB83P 36.6
66BG46AZ OWRYJXG9
66BG46AZ OWRYJXG9
success
success
success
